docs/topics/migrations.txt

```diff
@@ -276,6 +276,14 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+.. note::
```

Comment:
The current structure makes it look like this is a note under the `Model Managers` heading, which doesn't make sense. I think it should just be a regular paragraph under a new `Initial migrations` header, instead of a callout note.